website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -133,7 +133,7 @@ To leave an organization, click the ellipses (**...**) next to the organization
 Click **Password** in the sidebar to change your password. You cannot manage your HCP Terraform password directly if you linked your account to GitHub or HashiCorp Cloud Platform (HCP). Instead, manage your password through the linked service.
 
 -> **Note:** Password management is not available if your Terraform Enterprise instance uses [SAML single sign on](/terraform/enterprise/saml/configuration).
--> **Note:** Passwords must be at least 10 characters in length, and you can use any type of character. Password management is not available if your Terraform Enterprise instance uses [SAML single sign on](/terraform/enterprise/saml/configuration).
+-> **Note:** Passwords must be at least 8 characters long and contain at least 3 of the following: lowercase letters (a-z), uppercase letters (A-Z), numbers (0-9), special characters (!@#$%^&*). Password management is not available if your Terraform Enterprise instance uses [SAML single sign on](/terraform/enterprise/saml/configuration).
```

Comment:
Since this is a list of more than three, can we split this into a longer list:
```suggestion
<Note>

Passwords must be at least eight characters long, and contain at least three of the following:
- Lowercase letters (a-z)
- Uppercase letters (A-Z)
- Numbers (0-9)
- Special characters (!@#$%^&*)

Password management is not available if your Terraform Enterprise instance uses [SAML single sign on](/terraform/enterprise/saml/configuration).

</Note>
```

Reply:
That's a great idea! I implemented in the latest change 
<img width="714" height="560" alt="Screenshot 2025-08-28 at 2 47 57 PM" src="https://github.com/user-attachments/assets/d864539a-d69c-49c0-b601-f41286fc5812" />